CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
+- Fixed sending midi-off events from generals and another pushbuttons https://github.com/GrandOrgue/grandorgue/issues/1291
 # 3.9.2 (2022-11-29)
 - Fixed sending incorrect midi values from an enclosure https://github.com/GrandOrgue/grandorgue/issues/1267
-- Fixed not sending midi events from pushbuttons, including GC https://github.com/GrandOrgue/grandorgue/issues/1220
+- Fixed not sending midi events from GC https://github.com/GrandOrgue/grandorgue/issues/1220
```

Comment:
Even if the string was false, it's not recommended to change a release notes as it has been published.

Reply:
Fixed